inst/pages/04_containers.qmd

```diff
@@ -230,7 +230,7 @@ For information have a look at the [intro vignette](https://bioconductor.org/pac
  
    Option   Rows (features)    Cols (samples)               Recommended  
 ---------   --------------    ---------------  ------------------------
-   assays  	     match              match       Data transformations  
```

Comment:
Do these extra spaces do something?

Reply:
They don't, just looks more organized. I can remove them if you say?

Reply:
This renders a table view, might help with editing to have it visually organized like that in this case.

Reply:
Sure, edited it the same way again. I believe giving two extra spaces in row 2 aligns everything in that row (that's what I did in commit 3 - I had removed it in commit 2 earlier).

Reply:
@TuomasBorman This is ready to be merged ( I have made the requested changes, please check once if anything's there?)

Reply:
Just a gentle reminder :))